restaurants/views.py

```diff
@@ -69,3 +68,14 @@ def get(self, request, restaurant_id):
 
         except Restaurant.DoesNotExist:
             return JsonResponse({"message":"RESTAURANT_NOT_EXIST"}, status=404)        
+
+class RestaurantFoodsView(View):
+    def get(self, request, restaurant_id):
+        try:
+            foods      = Food.objects.filter(restaurant_id=restaurant_id)
+            foods_list = [{"id":f.id, "name":f.name, "price":f.price, "images":[i.image_url for i in f.images.all()]} for f in foods]
```

Comment:
변수를 줄여서 사용하는 방법은 코드의 가독성을 안좋게 만들기 때문에, 반드시 다른 사람이 코드를 읽어도 변수를 제대로 이해할 수 있도록 작성해 주세요~

```suggestion
            foods_list = [{
                "id"     : f.id, 
                "name"   : f.name, 
                "price"  : f.price, 
                "images" : [image.image_url for image in food.images.all()]
            } for food in foods]
```